config/manifests/patches/spec-descriptors.yaml

```
@@ -0,0 +1,452 @@
+      displayName: Rolling Restart Requested
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+    - path: canaryUpgrade
+      description: "Informs the operator to apply the requested changes to\na single rack. After manual validation set this value \nback to false and the operator will continue deployment.\n"
+      displayName: Canary Upgrade
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+    - path: canaryUpgradeCount
+      description: |
+        The number of nodes that will be updated when CanaryUpgrade
+        is true. Note that the value is either 0 or greater than the rack
+        size, then all nodes in the rack will get updated.
+      displayName: Canary Upgrade Count
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:podCount
+    - path: serverImage
+      description: |
+        Optional: Specify the name of the image to use for each
+        node in the cluster.
+      displayName: Server Image
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: systemLoggerImage
+      description: |
+        Container image for the log tailing sidecar container.
+      displayName: System Logger Image
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+    - path: configBuilderImage
+      description: "Optional: Specify the name of the image to use when \nrendering configuration files as each node in the \ncluster is initialized.\n"
+      displayName: Config Builder Image
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: configBuilderResource
+      description: "Resource requests and limits for each Cassandra pod in the \ncluster. Note: Only specify CPU and memory limits here.\n"
+      displayName: Config Builder Resources
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:resourceRequirements
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: replaceNodes
+      description: |
+        A list of pod names that need to be replaced
+      displayName: Replace Nodes
+    - path: replaceNodes[0]
+      description: |
+        A list of pod names that need to be replaced
+      displayName: Replace Nodes
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:text
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+    - path: disableSystemLoggerSidecar
+      description: |
+        Configuration for disabling the simple log tailing sidecar
+        container. Our default is to have it enabled.
+      displayName: Disable System Logger Sidecar
+      x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:booleanSwitch
+    - path: dockerImageRunsAsCassandra
+      description: |
+        Does the Server Docker image run as the Cassandra user?
```

This should include a bit more context, especially for the OpenShift UI.

```suggestion
        Indicates whether the Cassandra container runs as the built-in `cassandra` user. 
        On OpenShift clusters this field MUST be set to false given the random UID and GID
        assigned to each pod / container.
```